src/ApiClient/ServerClient.ts

```diff
@@ -303,6 +292,17 @@ export default new (class ServerClient extends ApiClient<IEvents> {
                 Api: `Tgstation.Server.Api/` + API_VERSION,
                 "Webpanel-Version": VERSION
             },
+            securityWorker: async () => {
```

Comment:
What decides if the securityWorker is used or not?

Reply:
if the request is "secured", eg on <https://tgstation.github.io/tgs-api-doc-host/#/> 

padlock ones require secured, which will trigger securityWorker

![image](https://user-images.githubusercontent.com/24603524/145671878-4d69de89-8574-4089-86be-b861078cabfc.png)